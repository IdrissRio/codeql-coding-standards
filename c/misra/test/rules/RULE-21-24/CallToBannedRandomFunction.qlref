rules/RULE-21-24/CallToBannedRandomFunction.ql