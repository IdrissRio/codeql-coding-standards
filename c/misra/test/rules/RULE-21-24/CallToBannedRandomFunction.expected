| test.c:5:3:5:7 | call to srand | Call to banned random number generation function 'srand'. |
| test.c:6:11:6:14 | call to rand | Call to banned random number generation function 'rand'. |
